hardware_accelerators/about-hardware-accelerators.adoc

```diff
@@ -0,0 +1,61 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="about-hardware-accelerators"]
+= About hardware accelerators
+include::_attributes/common-attributes.adoc[]
+:context: gpu-accelerators
+
+toc::[]
+
+Specialized hardware accelerators play a key role in the emerging generative artificial intelligence and machine learning (AI/ML) industry. Specifically, hardware accelerators are essential to the training and serving of large language and other foundational models that power this new technology. It allows data scientists, data engineers, ML engineers, and developers to take advantage of the specialized hardware acceleration for data-intensive transformations and model development and serving. Much of that ecosystem is open source with a number of contributing partners and open source foundations.
+
+Red{nbsp}Hat OpenShift provides support for cards and peripheral hardware that add processing units that comprise hardware accelerators:
```

Comment:
Remember to use the product title attribute rather than "OpenShift" in most cases.

Reply:
+1 to this comment from Max. Also, do you mean Red Hat OpenShift, or Red Hat OpenShift Container Platform?